src/components/Card.js

```diff
@@ -1,23 +1,36 @@
 export default class Card {
-  constructor({ name, link }, cardSelector, handleCardClick) {
+  constructor(
+    { name, link, isLiked, _id },
+    cardSelector,
+    popupSelector,
+    handleCardClick,
+    handleDelete,
+    handleToggleLikes
+  ) {
     this._name = name;
     this._link = link;
+    this._id = _id;
+    this._isLiked = isLiked;
     this._cardSelector = cardSelector;
+    this._popupSelector = popupSelector;
     this._handleCardClick = handleCardClick;
+    this._handleDelete = handleDelete;
+    this._handleToggleLikes = handleToggleLikes;
   }
   _setEventListeners() {
     //like btn
     this._cardElement
       .querySelector(".gallery__card-button")
```

Comment:
You've found the like button in line 69 and it's accessible here as `this._likeButton`

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.